fcitx-contrib/fcitx5-macos
# refactor file selector: allow cancel and replace same-name file

Test:
* Import a theme, apply, change some color, apply, export with the same name, import original theme, apply, should see original color.
* Import an image, apply, import another image with the same name, apply, should see new image.
* Import a CSS, apply, import another CSS with the same name, apply, should see new style.

Please update submodule on merge.
